scripts/firedrake-install

```diff
@@ -1113,6 +1145,8 @@ if mode == "install":
     else:
         git_clone("git+https://github.com/firedrakeproject/firedrake.git")
 
+    write_configuration(config)
```

Comment:
We do this early because now firedrake/setup.py depends on being able to import and read `firedrake_configuration`.